README.md

```diff
@@ -21,6 +21,11 @@ There are a few notable differences from [bevy_fly_camera](https://github.com/mc
 * Single-line setup
 * A tiny bit faster?
 
+Comparing [bevy_4x_camera](https://github.com/SarthakSingh31/bevy_4x_camera) :
```

Comment:
I don't think this comparison is necessary, it's a different type of camera and the choice should be somewhat obvious depending on your game's goals.